src/react/components/pages/homepage/homePage.tsx

```diff
@@ -139,11 +170,31 @@ export default class HomePage extends React.Component<IHomepageProps> {
     }
 
     private loadSelectedProject = async (project: IProject) => {
-        await this.props.actions.loadProject(project);
-        this.props.history.push(`/projects/${project.id}/edit`);
+        console.log(project);
```

Comment:
At this point the incoming project has already been converted and the confirmation should have already happened. Also - any need to bring the user to the project settings page vs. the edit page in this case?  The project has already been saved and they can easily go back to the project settings to make any additional changes.

Reply:
We had discussed earlier bringing them straight to the settings page so they can verify the info, given I've generated the titles of the project and its connections. But if we want to go straight to the edit page, I'd be fine with that and can get rid of the if clause

Reply:
We're creating new files anyway - so if there was an issue we already saved the file.  They can always re-open the v1 file vs. the newly generated v2 file.  I would keep the experience the same as v2.   If we encounter any errors/warnings during the conversion we should till display a modal with this failures.